.github/workflows/build.yml

```diff
@@ -1,346 +1,128 @@
-name: Build and Test
-
-
+name: Build and Publish
 
+#the concurrency check will cancel in-progress runs on non-master branches and pull requests on forked repositories, but not on the specified branches (master, github-action-**, DAT-**, dependabot-**).
 concurrency:
-  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
-  cancel-in-progress: true
+  group: ${{ github.workflow }}-${{ github.head_ref && github.ref || github.run_id }}
+  cancel-in-progress: false
```

Comment:
Is cancel-in-progress supposed to be false?

Reply:
I will check on this again. I remember I had to do this for testing.